StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -90,11 +90,21 @@ class NativeFlowController {
                 || navigationController.topViewController is NetworkingLinkVerificationViewController
                 || navigationController.topViewController is NetworkingSaveToLinkVerificationViewController
                 || navigationController.topViewController is LinkAccountPickerViewController)
-        closeAuthFlow(
-            showConfirmationAlert: showConfirmationAlert,
-            showNetworkingLanguageInConfirmationAlert: (dataManager.manifest.isNetworkingUserFlow == true && navigationController.topViewController is NetworkingLinkSignupViewController),
-            error: nil
-        )
+
+        let finishClosingAuthFlow = { [weak self] in
+            self?.closeAuthFlow()
+        }
+        if showConfirmationAlert {
+            CloseConfirmationAlertHandler.present(
+                businessName: dataManager.manifest.businessName,
+                showNetworkingLanguageInConfirmationAlert: (dataManager.manifest.isNetworkingUserFlow == true && navigationController.topViewController is NetworkingLinkSignupViewController),
+                didSelectOK: {
+                    finishClosingAuthFlow()
```

Comment:
my kotlin brain is telling me to inline this in 102 and 106, but not sure if this is a swift custom, so will let you decide :)

Reply:
Yea I'm probably 'overthinking' this a little bit but the basic fear/thought is that the intention is that whether we enter `showConfirmationAlert`, or not, the logic that is executed should be the same. If we do not put this in a shared closure/function, then there's a non-zero chance that it gets out of sync because its updated in one place, but not the other.

Reply:
In a wild twist of faith: I actually ended up adding extra logic to `finishClosingAuthFlow` in a future-PR-I'm-making so it worked out!